test/index.js

```diff
@@ -34,17 +36,121 @@ describe('Wigzo', function () {
   describe('Constructing the integration', function () {
     it('should have the right settings', function () {
       analytics.compare(Wigzo, Integration('Wigzo')
+        .readyOnInitialize()
+        .global('wigzo')
+        .option('licenseCode', ''));
     })
-  })
 
-  describe('#initialize', function () {
-    it('should call this.ready() with valid options', function () {
-      analytics.stub(wigzo, 'ready')
-      wigzo.initialize()
-      analytics.called(wigzo.ready)
-    })
+    describe('#initialize', function () {
+      it('should call this.ready() with valid options', function () {
+        analytics.stub(wigzo, 'ready')
+        wigzo.initialize()
+        analytics.called(wigzo.ready)
+      })
+
+      it('should create window.wigzo', function() {
+        analytics.assert(window.wigzo);
+      });
+
+      it('should create window.wigzo.track', function() {
+        analytics.assert(window.wigzo.track);
+      });
+
+      it('should create window.wigzo.index', function() {
+        analytics.assert(window.wigzo.index);
+      });
 
-    it('should prepare the needed globals', function () {
+      it('should create window.wigzo.page', function() {
+        analytics.assert(window.wigzo.page);
+      });
     })
   })
+
+  describe('loading', function() {
+    it('should load', function(done) {
+      analytics.load(wigzo, done);
+    });
+  });
+
+  describe('after loading', function() {
+    beforeEach(function(done) {
+      analytics.initialize();
+      analytics.page();
+      analytics.once('ready', done);
+    });
+
+    describe('#page', function() {
+      beforeEach(function() {
+        analytics.stub(window.wigzo, 'page');
+      });
+
+      it('should call page', function() {
+        analytics.page();
+        analytics.called(window.wigzo.page);
+      });
+
+      it('should pass page name and default properties via page', function() {
+        analytics.page('Name');
+        analytics.called(window.wigzo.page, 'Name', {
+          title: document.title,
+          url: window.location.href,
+          name: 'Name',
+          path: window.location.pathname,
+          referrer: document.referrer,
+          search: window.location.search
+        });
+      });
+    });
+
+    describe('#identify', function() {
+      beforeEach(function() {
+        analytics.stub(window.wigzo.identify, 'identify');
+      });
+
+      it('should send an id', function() {
+        analytics.identify('id');
+        analytics.called(window.wigzo.identify, 'id');
+      });
+
+      it('should send traits', function() {
+        analytics.identify({
+          trait: true
+        });
+        analytics.called(window.wigzo.identify, {
+          trait: true
```

Comment:
again i dont think this should pass. you are not allowing custom traits to pass through in your integration code. it seems like you are only picking out name, email, and phone